### PR TITLE
New packages: SIL fonts

### PR DESCRIPTION
**Pull request by sirikid**
> [SIL International](https://www.sil.org/) is a faith-based nonprofit organization committed to serving language communities worldwide as they build capacity for sustainable language development.

So I wrote templates for their fonts.

### CODE REVIEW

File: srcpkgs/font-sil-abyssinica/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'font-sil-abyssinica'
+pkgname=font-sil-abyssinica
+version=1.500
+revision=1
+archs=noarch
+wrksrc=AbyssinicaSIL-$version
+_sil_font_dir=/usr/share/fonts/SIL
+make_dirs="$_sil_font_dir 0755 root root"
```
Comment (maxice8) [inline srcpkgs/font-sil-abyssinica/template]: This is not necessary

File: srcpkgs/font-sil-abyssinica/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'font-sil-abyssinica'
+pkgname=font-sil-abyssinica
+version=1.500
+revision=1
+archs=noarch
+wrksrc=AbyssinicaSIL-$version
```
Comment (maxice8) [inline srcpkgs/font-sil-abyssinica/template]: quote this

File: srcpkgs/font-sil-abyssinica/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+_sil_font_dir=/usr/share/fonts/SIL
```
Comment (maxice8) [inline srcpkgs/font-sil-abyssinica/template]: this should be the variable font_dirs=

File: srcpkgs/font-sil-abyssinica/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+make_dirs="$_sil_font_dir 0755 root root"
+hostmakedepends="unzip"
+depends="font-util"
+short_desc="Ethiopic font, based on Ethiopic calligraphic traditions"
+maintainer="Ivan Sokolov <ivan-p-sokolov@ya.ru>"
+license="OFL-1.1"
+homepage="https://software.sil.org/abyssinica/"
+distfiles="https://software.sil.org/downloads/r/abyssinica/$wrksrc.zip"
```
Comment (maxice8) [inline srcpkgs/font-sil-abyssinica/template]: use only $version

**Comment by sirikid** [comment]
@maxice8 I edited [the last template](https://github.com/void-linux/void-packages/pull/11108/commits/fd2d43ab25bf95d724750273ae292cf905d467e3), if it's fine then I'll push the rest.

**Comment by sirikid** [comment]
Thank you for your time!

**Pull request merged**